nodes/persistence-node/src/config/IpfsConfig.ts

```diff
@@ -1,8 +1,19 @@
+import { isValidUrl } from "../isValidUrl";
+
 export class IpfsConfig {
   ipfsProvider = process.env.IPFS_PROVIDER ?? "http://localhost:5001";
+  ipfsUrl = ensureValidUrl(process.env.IPFS_URL);
   gatewayURI = process.env.IPFS_GATEWAY ?? "https://ipfs.io/ipfs";
   objectGetTimeout = parseInt(process.env.IPFS_OBJECT_GET_TIMEOUT!) ?? 15000;
   pinTimeout = parseInt(process.env.IPFS_PIN_TIMEOUT!) ?? 30000;
   unpinTimeout = parseInt(process.env.IPFS_UNPIN_TIMEOUT!) ?? 30000;
   gatewayTimeout = parseInt(process.env.IPFS_GATEWAY_TIMEOUT!) ?? 15000;
 }
+
+function ensureValidUrl(url: string | undefined) {
```

Comment:
A better name would be "getValidUrlOrUndefined" or "validUrlOrDefined". I saw the same convention before. Using "ensureValidUrl" is a bit confusing, it's not obvious to the reader how something can ensure a valid url